### PR TITLE
feat(interactive-btn): add interactive button style class

### PR DESCRIPTION
**Pull request by ranst91-u**
Preview: https://static-staging.usabilla.com/stylabilla/pr/blup-499-subtle-interactive-btn/docs/section-components.html#kssref-components-button-colors

## What to test? Scope of the test?

Add "interactive button" style. Which should currently by used by several subtle buttons

## Reviews

Reviewers: @chiel @MLKiiwy @thefrontendwizard @kagnamazian @realdreamer @DavideBruner @ranst91-u @Maruceau @gsbkr 
Required: 1 FE approval - 1 UI approval

### CODE REVIEW

File: src/components/_button.scss [hunk 1]
```diff
@@ -216,6 +216,7 @@ $sb-btn-link-padding: sb-rem-spacing('xs');
 // .sb-btn--danger - Danger button
 // .sb-btn--warning - Warning button
 // .sb-btn--success - Success button
+// .sb-btn--interactive - Success button
```
Comment (realdreamer) [inline src/components/_button.scss]:
```suggestion
// .sb-btn--interactive - Interactive button
```

maybe primary outline something like that, all that buttons are interactive right!

Comment (ranst91-u) [reply src/components/_button.scss]: When I talked to @Maruceau that was the name it was given.
I agree we might need to discuss the naming and decide. But it's defintely not primary outline. It wlll currently by applied to some subtle buttons

**Comment by ranst91-u** [comment]
@realdreamer @Maruceau 
I resolved Mohamed's suggestion, however, if the naming discussion should still take place, I suggest we have it.
Currently, the name we use for this is "interactive", to differentiate it from normal subtle buttons (this currently applied to subtle buttons). Let me know if anything needs changing or if I should give it a stronger binding to the subtle button

**Comment by Maruceau** [comment]
I see this is merged already, so I don't if my opinion still matters. But why do we need this? All buttons are interactive so I don't understand why we have a separate button called interactive.

**Pull request merged**